src/components/common/NewActionButton/NewActionButton.tsx

```diff
@@ -87,172 +95,173 @@ const NewActionButton = (/* { colony, ethDomainId }: Props */) => {
-  //       nextStepEdit: 'dashboard.EditDomainDialog',
-  //       nextStepManageWhitelist: 'dashboard.ManageWhitelistDialog',
-  //       colony,
-  //     },
-  //   },
-  //   {
-  //     component: CreateDomainDialog,
-  //     props: {
-  //       prevStep: 'dashboard.ManageDomainsDialog',
-  //       colony,
-  //     },
-  //   },
-  //   {
-  //     component: EditDomainDialog,
-  //     props: {
-  //       prevStep: 'dashboard.ManageDomainsDialog',
-  //       colony,
-  //       ethDomainId,
-  //     },
-  //   },
-  //   {
-  //     component: ManageReputationDialog,
-  //     props: {
-  //       nextStepAwardReputation: 'dashboard.AwardDialog',
-  //       nextStepSmiteReputation: 'dashboard.SmiteDialog',
-  //       prevStep: 'dashboard.ColonyActionsDialog',
-  //       colony,
-  //     },
-  //   },
-  //   {
-  //     component: AwardDialog,
-  //     props: {
-  //       prevStep: 'dashboard.ManageReputationDialog',
-  //       colony,
-  //       ethDomainId,
-  //     },
-  //   },
-  //   {
-  //     component: SmiteDialog,
-  //     props: {
-  //       prevStep: 'dashboard.ManageReputationDialog',
-  //       colony,
-  //       ethDomainId,
-  //     },
-  //   },
-  //   {
-  //     component: AdvancedDialog,
-  //     props: {
-  //       prevStep: 'dashboard.ColonyActionsDialog',
-  //       nextStepPermissionManagement: 'dashboard.PermissionManagementDialog',
-  //       nextStepRecovery: 'dashboard.RecoveryModeDialog',
-  //       nextStepEditDetails: 'dashboard.EditColonyDetailsDialog',
-  //       nextStepVersionUpgrade: 'dashboard.NetworkContractUpgradeDialog',
-  //       colony,
-  //     },
-  //   },
-  //   {
-  //     component: PermissionManagementDialog,
-  //     props: {
-  //       prevStep: 'dashboard.AdvancedDialog',
-  //       colony,
-  //       ethDomainId,
-  //     },
-  //   },
-  //   {
-  //     component: RecoveryModeDialog,
-  //     props: {
-  //       prevStep: 'dashboard.AdvancedDialog',
-  //       colony,
-  //     },
-  //   },
-  //   {
-  //     component: ManageWhitelistDialog,
-  //     props: {
-  //       prevStep: 'dashboard.ManageDomainsDialog',
-  //       colony,
-  //     },
-  //   },
-  //   {
-  //     component: NetworkContractUpgradeDialog,
-  //     props: {
-  //       prevStep: 'dashboard.AdvancedDialog',
-  //       colony,
-  //     },
-  //   },
-  //   {
-  //     component: EditColonyDetailsDialog,
-  //     props: {
-  //       prevStep: 'dashboard.AdvancedDialog',
-  //       colony,
-  //     },
-  //   },
-  //   {
-  //     component: TokenMintDialog,
-  //     props: {
-  //       prevStep: 'dashboard.ManageFundsDialog',
-  //       colony,
-  //     },
-  //   },
-  //   {
-  //     component: ColonyTokenManagementDialog,
-  //     props: {
-  //       prevStep: 'dashboard.ManageFundsDialog',
-  //       colony,
-  //     },
-  //   },
-  // ]);
-  const startWizardFlow = (...args) => args;
+  const startWizardFlow = useNaiveBranchingDialogWizard([
```

Comment:
Perhaps have this live in a separate config file? I wouldn't say it needs to live inside the component like the one in `ColonyActionsDialog` as it's only receiving a couple of props (colony and domainId) that can be passed as arguments to the wrapper function (e.g. getWizardFlow, which would return `startWizardFlow` for use in the click handler).

Reply:
Moved the flow array to its own config file inside the folder